package.json

```diff
@@ -69,6 +69,7 @@
     "validate-declarations": "babel-node --extensions \".ts\" scripts/validateTypescriptDeclarations.ts"
   },
   "devDependencies": {
+    "@argos-ci/core": "^0.3.0",
```

Comment:
@michaldudak a new version 0.4.0 has been released with a better compression algorithm. Could you please upgrade?

Also we have a Discord, feel free to join! https://discord.gg/vMyXWVA8

Reply:
I'll try it right away, thanks.

Reply:
The build still fails the same way. It doesn't seem to depend on the version of CLI/SDK used.